elephant/sta.py

```diff
@@ -314,3 +320,57 @@ def spike_field_coherence(signal, spiketrain, **kwargs):
 
     return (pq.Quantity(sfc, units=pq.dimensionless),
             pq.Quantity(frequencies, units=pq.Hz))
+
+
+def stPR(spiketrains, bin_size=1*pq.ms, sigma=5*pq.ms):
+    """
+    Spike triggered population response as described in [1].
+
+    Parameters
+    ----------
+    spiketrains : list of neo.SpikeTrains
+        list of spike train objects containing a list of spiketimes. The spike
+        trains should have the same t_start and t_stop times.
+
+    bin_size : pq.Quantity
+        Time quantity for binning of the `spiketrains`.
+        Default: 1*pq.ms
+
+    sigma : pq.Quantity
+        Standard deviation of gaussian kernel for estimation of population
+        rate.
+        Default: 5*pq.ms
+
```

Comment:
After Parameters, before References, add Returns statement
```
list of neo.AnalogSignal
    (description)
```

Reply:
Hi,
I intentionally excluded this one, because probably a list of analogsignals is not the ideal output. Do you think that a list of pq.Quantity is a better option? I am not sure if the units of the stPR are meaningful in any way, maybe just floats can also do the job?

Reply:
I'll discuss it with someone else.

Reply:
Alright, we agreed on the following:
* remove with_nan
* if n_spikes < 2:
    warning.warn(blah-blah)
    return np.nan

because if the num. of spikes is less than 2, it does not make sense to compute CCH anyway.